README.md

```diff
@@ -59,63 +59,78 @@ on any platform.
 
 Before starting, install Docker ([mac], [windows], [linux]) and [Docker Compose].
 
-After installing,
-pull [the application image] from Docker Hub,
-or build it yourself.
-All of the dependencies are managed through Docker,
-so there shouldn't be any problems running this script.
-If you do encounter a problem, please [open an issue] to report it.
+#### Mac or Windows VM setup
+
+If you are running Docker for the first time on OS X or Windows,
+you'll need to [install docker-machine] and [VirtualBox],
+then create a virtual machine:
+
+```bash
+$ docker-machine create --driver virtualbox default
+
+# Add relevant environment variables to your shell
+# (if you don't use bash, replace with `.zshrc`, `.profile`, etc.
+$ cat 'eval $(docker-machine env)' >> ~/.bashrc && source ~/.bashrc
+
+# Open up port 3000 between the host and the virtualbox machine
+$ VBoxManage modifyvm "default" --natpf1 "guestnginx,tcp,,3000,,3000"
```

Comment:
This is a command-line way to open up port 3000. You did it through the VirtualBox GUI, but this is easier to explain in a README.